spex/scripts/spex-adapt-commands.sh

```
@@ -0,0 +1,163 @@
+  echo "ERROR: Malformed JSON in $MAPPING_TABLE" >&2
+  exit 1
+fi
+
+TMPDIR_WORK=$(mktemp -d "${TMPDIR:-/tmp}/spex-adapt-XXXXXX")
+trap 'rm -rf "$TMPDIR_WORK"' EXIT
+
+FALLBACK_NOTE=$(jq -r '.fallback_note // ""' "$MAPPING_TABLE")
+HARNESS_ID=$(jq -r '.harness // ""' "$MAPPING_TABLE")
+
+# Pre-extract inline substitutions into a single TSV file for batched processing
+INLINE_COUNT=$(jq '.inline // [] | length' "$MAPPING_TABLE")
```

Shellcheck (0.11.0)

[warning] INLINE_COUNT appears unused. Verify use (or export if used externally).

(SC2034)